### ISSUE TITLE
buttersink depends on python-psutils and python-crcmod but does not install it

### ISSUE THREAD

**Issue by a3nm** [AmesCornish/buttersink#24]
buttersink installs various dependencies when you do make, but when I run it afterwards I get: 

```
Traceback (most recent call last):
  File "./buttersink.py", line 22, in <module>
    import ButterStore
  File "/home/a3nm/apps/buttersink/buttersink/ButterStore.py", line 10, in <module>
    import Butter
  File "/home/a3nm/apps/buttersink/buttersink/Butter.py", line 13, in <module>
    import psutil
ImportError: No module named psutil
```

I think python-psutil should be added to apt.txt. Likewise, python-crcmod should be added as a dependency.

**Comment by AmesCornish** [comment]
Those two dependencies are listed in pip.txt, and make should install them.  If you `make clean` and then `make`, does it not pip install crcmod and psutil?

**Comment by a3nm** [comment]
Thanks for your answer! It turns out that pip indeed installed the packages in `/usr/local/lib/python2.7/dist-packages` but they were only readable by root because my umask is 077 and this is preserved by sudo...

This could be fixed by ensuring that the umask is 022 when running the sudo pip install command in the makefile, following https://unix.stackexchange.com/a/65890

**Comment by AmesCornish** [comment]
Fix in 0.6.8
